.nsm.mk

```diff
@@ -16,14 +16,14 @@
 # and docker build infrastructure. It also contains the targets to build
 # and push Docker images
 
-DOCKER_NETMESH_TEST=networkservicemesh/netmesh-test
-DOCKER_NETMESH=networkservicemesh/netmesh
-DOCKER_TEST_DATAPLANE=networkservicemesh/test-dataplane
-DOCKER_NSM_INIT=networkservicemesh/nsm-init
-DOCKER_NSE=networkservicemesh/nse
-DOCKER_RELEASE=networkservicemesh/release
-DOCKER_SIDECAR_INJECTOR=networkservicemesh/sidecar-injector
-DOCKER_SRIOV_CONTROLLER=networkservicemesh/sriov-controller
+ORG=networkservicemesh
+
+.PHONY: docker-build-%
+docker-build-%: docker-build-release
+	@${DOCKERBUILD} -t ${ORG}/$* -f build/Dockerfile.$* .
+	@if [ "x${COMMIT}" != "x" ] ; then \
+		docker tag ${ORG}/$* ${ORG}/$*:${COMMIT} ;\
```

Comment:
See line 40

Reply:
See response to line 40 :)  Also... I found it that way :)